src/oss/confluent.sh

```diff
@@ -1355,13 +1390,26 @@ These are the available commands:
 
     config      Configure a connector.
 
+EOF
+for ((i = 0; i < ${#dynamic_cli_commands[@]}; ++i)); do
```

Comment:
Make the usage message generic would be good. Still we would have to account for the right widths (virtual tab spaces) depending on the length of the command. For instance `log` needs more tabs than `config`. We'll need some arithmetic there. Given that we might want to just add the raw line here. Or leave the loop knowing that it doesn't generalize as is.